commit 6584ee2c7adbf63ba426682a77da4341f65ecf88
Author: NO NAME <NO EMAIL>
Date:   Fri Mar 31 05:30:03 2017 -0500

    Increase memory limit for genautomata on AIX.
    
    When doing 64-bit builds of GCC on AIX, genautomata experiences malloc()
    failures. It seems that a 512MB heap is no longer big enough.
    
    I also updated the comments in this file to match the values passed to the
    linker. The AIX `ld` manual says this about the maxdata option:
    
        Sets the maximum size (in bytes) that is allowed for the user data area
        (or user heap) when the executable program is run.
    
    So the comments were giving incorrect values.
    
    gcc/
    
      * config/rs6000/x-aix: Increase memory limit for genautomata on AIX.
      It is now experiencing malloc() failures during 64-bit compiler builds.
      Also fix comments.

diff --git a/gcc/config/rs6000/x-aix b/gcc/config/rs6000/x-aix
index d40690f2da6..da465671ee6 100644
--- a/gcc/config/rs6000/x-aix
+++ b/gcc/config/rs6000/x-aix
@@ -1,6 +1,6 @@
-# genautomata requires more than 256MB of data
-build/genautomata : override LDFLAGS += -Wl,-bmaxdata:0x20000000
+# genautomata requires more than 1GB of data
+build/genautomata : override LDFLAGS += -Wl,-bmaxdata:0x40000000
 
-# jc1 requires more than 256MB of data
+# jc1 requires more than 1GB of data
 $(COMPILERS) : override LDFLAGS += -Wl,-bmaxdata:0x40000000
 
